DOCS/interface-changes.rst

```diff
@@ -30,6 +30,8 @@ Interface changes
 ::
 
  --- mpv 0.38.0 ---
+    - add `context-menu` command
```

Comment:
Move those to `DOCS/interface-changes` using new policy https://github.com/mpv-player/mpv/blob/master/DOCS/contribute.md#interface-change-policy